synonyms/synonyms.js

```diff
@@ -0,0 +1,38 @@
+import { createDirectus, readItems, rest, staticToken } from "@directus/sdk";
+import words from "an-array-of-english-words" assert { type: "json" };
+import "dotenv/config.js";
+import * as fs from "fs";
+const getDirectusClient = createDirectus(process.env.DIRECTUS_URL)
+  .with(staticToken(process.env.DIRECTUS_STATIC_TOKEN))
+  .with(rest());
+
+const audioData = await getDirectusClient.request(
+  readItems("audios", {
+    fields: ["id", "title", "topics"],
+    filter: { status: { _eq: "active" } },
+  })
+);
+
+let cmsData = "";
+Object.values(audioData).forEach((value) => {
+  const title = value["title"];
+  const topics = value["topics"];
+  const allWords = title.concat(topics);
```

Comment:
Be careful here, you can merge the last and the first words into one. I'd better process each string separately and then merge the words list.